### ISSUE TITLE
test flops and parameters 

### ISSUE THREAD

**Issue by luoyinTT** [open-mmlab/mmediting#306]
when i try to use the tools/get_flops.py to compute flops and parameters follow the guide , i met

```bash
(open-mmlab) yinl@xcdl190206:/scratch/workspace/yinl/mmediting-master$ python tools/get_flops.py configs/restorers/basicvsr/basicvsr_vimeo90k_bi.py --shape 180 120 
2021-05-14 10:39:09,790 - mmedit - INFO - Use load_from_local loader
lrs.size(): torch.Size([1, 3, 180, 120])
Traceback (most recent call last):
  File "tools/get_flops.py", line 57, in <module>
    main()
  File "tools/get_flops.py", line 47, in main
    flops, params = get_model_complexity_info(model, input_shape)
  File "/scratch/workspace/yinl/anaconda3/envs/open-mmlab/lib/python3.7/site-packages/mmcv/cnn/utils/flops_counter.py", line 104, in get_model_complexity_info
    _ = flops_model(batch)
  File "/scratch/workspace/yinl/anaconda3/envs/open-mmlab/lib/python3.7/site-packages/torch/nn/modules/module.py", line 889, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/var/lib/docker/scratch/workspace/yinl/mmediting-master/mmedit/models/restorers/basic_restorer.py", line 174, in forward_dummy
    out = self.generator(img)
  File "/scratch/workspace/yinl/anaconda3/envs/open-mmlab/lib/python3.7/site-packages/torch/nn/modules/module.py", line 889, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/var/lib/docker/scratch/workspace/yinl/mmediting-master/mmedit/models/backbones/sr_backbones/basicvsr_net.py", line 117, in forward
    n, t, c, h, w = lrs.size()
ValueError: not enough values to unpack (expected 5, got 4)
```

how to solve it?

**Comment by ckkelvinchan** [comment]
Since BasicVSR takes a video (instead of an image) as input, the existing get_flops.py may be incompatible. I will look into the problem and get back to you soon. Thanks for reporting.

**Comment by ckkelvinchan** [comment]
Fixed in #309. Please reopen if there are any further problems.

**Comment by valleballe** [comment]
I have a similar issue running restoration_video_demo.py 

I followed the installation instructions:
```
pip3 install openmim
mim install mmcv-full
git clone https://github.com/open-mmlab/mmediting.git
cd mmediting
pip3 install -e .
```
And got the following error:
````
python demo/restoration_video_demo.py \
  /content/mmediting/configs/restorers/real_basicvsr/realbasicvsr_c64b20_1x30x8_lr5e-5_150k_reds.py \
  https://download.openmmlab.com/mmediting/restorers/real_basicvsr/realbasicvsr_c64b20_1x30x8_lr5e-5_150k_reds_20211104-52f77c2c.pth \
  /content/input \
  /content/output


/usr/local/lib/python3.8/dist-packages/mmcv/__init__.py:20: UserWarning: On January 1, 2023, MMCV will release v2.0.0, in which it will remove components related to the training process and add a data transformation module. In addition, it will rename the package names mmcv to mmcv-lite and mmcv-full to mmcv. See https://github.com/open-mmlab/mmcv/blob/master/docs/en/compatibility.md for more details.
  warnings.warn(
2023-01-06 17:52:29,160 - mmedit - INFO - load checkpoint from http path: https://download.openmmlab.com/mmediting/restorers/basicvsr/spynet_20210409-c6c1bd09.pth
2023-01-06 17:52:31,203 - mmedit - INFO - load checkpoint from torchvision path: torchvision://vgg19
load checkpoint from http path: https://download.openmmlab.com/mmediting/restorers/real_basicvsr/realbasicvsr_c64b20_1x30x8_lr5e-5_150k_reds_20211104-52f77c2c.pth
Traceback (most recent call last):
  File "demo/restoration_video_demo.py", line 90, in <module>
    main()
  File "demo/restoration_video_demo.py", line 66, in main
    output = restoration_video_inference(model, args.input_dir,
  File "/content/mmediting/mmedit/apis/restoration_video_inference.py", line 116, in restoration_video_inference
    result = model(
  File "/usr/local/lib/python3.8/dist-packages/torch/nn/modules/module.py", line 1190, in _call_impl
    return forward_call(*input, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/mmcv/runner/fp16_utils.py", line 119, in new_func
    return old_func(*args, **kwargs)
  File "/content/mmediting/mmedit/models/restorers/srgan.py", line 95, in forward
    return self.forward_test(lq, gt, **kwargs)
  File "/content/mmediting/mmedit/models/restorers/real_esrgan.py", line 212, in forward_test
    output = _model(lq)
  File "/usr/local/lib/python3.8/dist-packages/torch/nn/modules/module.py", line 1190, in _call_impl
    return forward_call(*input, **kwargs)
  File "/content/mmediting/mmedit/models/backbones/sr_backbones/real_basicvsr_net.py", line 67, in forward
    n, t, c, h, w = lqs.size()
ValueError: not enough values to unpack (expected 5, got 4)
````

[Here is a colab notebook for replication](https://colab.research.google.com/drive/1CNUMoMO5lnsK5AHju4TYYIf_gLryqZiP?usp=sharing)
